### PR TITLE
added NppFTP v0.27.5

### PR DESCRIPTION
**Pull request by chcg**
added https://github.com/ashkulz/NppFTP/releases/tag/v0.27.5

### CODE REVIEW

**Comment by donho** [comment]
@chcg Thank you for your effort.
I have just added sha-256 implementation in Notepad++, and it will be applied to Plugin Admin instead of MD5. All the doc and binaries are updated on:
https://notepad-plus-plus.org/features/plugin-admin.html
Could you change MD5 hash to SHA-256 in the list and download/check with the new notepad++ in debug mode ?

File: src/pl.x64.json [hunk 1]
```diff
@@ -62,6 +62,16 @@
 			"description": "This template is done for making plugin development as easy and simple as possible. 4 steps and it's done - Even your grandmom can do it!",
 			"author":"Don HO",
 			"homepage": "https://notepad-plus-plus.org/contribute/plugin-howto.html"
+		},
+		{
+			"folder-name": "NppFTP",
+			"display-name": "NppFTP",
+			"version": "0.27.5",
+			"id": "75bbe9f9867493d7b042539028111aef",
```
Comment (donho) [inline src/pl.x64.json]: Now it's sha-256 hash

File: src/pl.x86.json [hunk 2]
```diff
@@ -62,6 +62,16 @@
 			"description": "This template is done for making plugin development as easy and simple as possible. 4 steps and it's done - Even your grandmom can do it!",
 			"author":"Don HO",
 			"homepage": "https://notepad-plus-plus.org/contribute/plugin-howto.html"
+		},
+		{
+			"folder-name": "NppFTP",
+			"display-name": "NppFTP",
+			"version": "0.27.5",
+			"id": "0435f78c39a294e6a23056625085bfb5",
```
Comment (donho) [inline src/pl.x86.json]: sha-256 hash

**Comment by chcg** [comment]
Done. Added sha-256.

There is a typo in  https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp#L547

				TEXT("Plugin ID missmathed"),

should be mismatched.

**Comment by donho** [comment]
@chcg Have you check your modification with new binary? the 32-bit doesn't work.
And obviously, The hash is not correct.

**Comment by chcg** [comment]
Just the 64 bit version and I needed to take the appveyor debug build of n++ from commit
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/519fecbb827eb011e6da6d546e5da1d22fb982a8

, because the linked zip at https://notepad-plus-plus.org/pluginListTestTools/notepad++7.5.9.debug.x64.zip

Yesterday was still the one with md5.

**Comment by chcg** [comment]
@donho The n++ files are now ok. Seems some local build got in my way for x86 from which i created the erroneous sha checksum. Sorry for that. Thanks for finding it.

**Pull request closed without merging**